packages/core/src/util/address.ts

```diff
@@ -0,0 +1,353 @@
+import { CONST, wallet } from "@cityofzion/neon-core-next";
+import * as Base58Check from "base58check";
+import * as bech32 from "bech32";
+import * as BIP32 from "bip32";
+import * as BIP39 from "bip39";
+import CryptoJS from "crypto-js";
+import { HDNodeWallet, Mnemonic, SigningKey, computeAddress, keccak256, ripemd160, sha256 } from "ethers";
+import * as secp256k1 from "secp256k1";
+import * as secp256r1 from "secp256r1";
+import { Bech32AddrType, BIP44_PURPOSE, NEO_COIN_TYPE, ETH_COIN_TYPE, SWTH_COIN_TYPE, Network, defaultNetworkConfig } from "../env";
+import { BIP44Path, randomMnemonic, stringOrBufferToBuffer, stripHexPrefix } from "./crypto";
+
+export const ZeroAddress = "0x0000000000000000000000000000000000000000";
+
+export interface AddressOptions {}
+
+export interface AddressBuilder<T extends AddressOptions = AddressOptions> {

```

Comment:
we're not bring this functionality over as well. can you reference the constructor of `DemexWallet` for examples of address conversion?